coq_tools/find_bug.py

```diff
@@ -2237,20 +2237,30 @@ def make_make_coqc(coqc_prog, **kwargs):
                 )
                 coq_theories_path = os.path.join(coq_lib_path, "theories")
                 coq_user_contrib_path = os.path.join(os.path.join(coq_lib_path, "user-contrib"), "Stdlib")
+                try: coqpath_paths = os.environ.get('COQPATH').split(':')
```

Comment:
```suggestion
                coqpath_paths = os.environ.get('COQPATH', '').split(':')
```
is a bit more concise and perhaps a bit more idiomatic

Reply:
Thanks, as you can see I'm definitely no pytohn expert ;-)